packages/api/src/graphql/query.ts

```diff
@@ -902,6 +946,26 @@ export const GraphQLPublicQuery = new GraphQLObjectType<undefined, Context>({
           limit: Limit(first, last)
         })
       }
+    },
+
+    // Comments
+    // =======
+    comments: {
```

Comment:
Why would you need that in the public API?

Reply:
I think we will need for the paginated comments lists, what do you think?

Reply:
At the moment you only can get comments through the article. So you don't need that query here. You need to enhance the `article` query in the public query

Reply:
removed this and will add pagination in another PR